h/static/styles/partials-v2/_search-bar.scss

```diff
@@ -60,7 +63,8 @@
   display: none;
   padding: 20px 0 0 0;
   position: absolute;
-  margin-top: 52px;
+  top: 52px;
```

Comment:
Ah OK. I see the same in Firefox. I don't suppose you know why there is a difference between Chrome and Safari/Firefox here?